src/components/UserNicknamesPanel/UserNicknamesPanel.module.scss

```diff
@@ -21,6 +21,11 @@
   span {
     word-break: break-word;
   }
+
+  div {
+    display: flex;
+    gap: 0.5rem;
+  }
```

Comment:
we should avoid adding further structure-bound styles and instead define this as it's own style at the top level of the file. maybe something like `.controlContainer`?